### PR TITLE
add support for ibm-semeru-21

### PR DESCRIPTION
**Pull request by rmie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,6 +232,7 @@ Some come from the parent images and some are installed in this image for backwa
 | eclipse-temurin-21-alpine     |     | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    | ✔️      |     |     |
 | ibm-semeru-11-focal           | ✔️   | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    | ✔️      |     |     |
 | ibm-semeru-17-focal           | ✔️   | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    | ✔️      |     |     |
+| ibm-semeru-21-focal           | ✔️   | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    | ✔️      |     |     |
```
Comment (carlossg) [inline README.md:235]: wrong distro?

Comment (rmie) [reply README.md:235]: good catch, fixed

**Comment by rmie** [comment]
> there are some extra files missing like the github action, can you check https://github.com/carlossg/docker-maven?tab=readme-ov-file#adding-new-images

I think I got them all, this time. _common.sh_ already contains "ibm-semeru" so "ibm-semeru-21-jammy" should be covered.

**Comment by carlossg** [comment]
thanks!

**Pull request merged**